### PR TITLE
IMPRO-1534: convert extend radar mask to use clize

### PR DESCRIPTION
**Pull request by neilCrosswaite**
# Convert radar mask to use clize
- The order that the arguments go into the cli are not the other way around.

Testing:
 - [x] Ran tests and they passed OK

### CODE REVIEW

**Comment by arh89** [comment]
@neilCrosswaite should this go into the feature branch?

File: lib/improver/cli/extend_radar_mask.py [hunk 1]
```diff
@@ -30,46 +30,15 @@
...
-    save_netcdf(remasked_data, args.output_filepath)
-
-
-def process(coverage, radar_data, fix_float64=False):
+@cli.clizefy
+@cli.with_output
+def process(coverage: cli.inputcube,
+            radar_data: cli.inputcube,
+            *,
+            fix_float64=False):
```
Comment (cgsandford) [inline lib/improver/cli/extend_radar_mask.py]: Nice work but could you try removing this option please?  The `ExtendRadarMask` plugin should be outputting a float32 cube, so there should be no need to call `check_cube_not_float64` at line 66 (below).

**Comment by cgsandford** [comment]
@arh89 we agreed the nowcasting CLIs could go straight into master to allow the feature branch to be sorted ASAP.  I don't think this one really matters as it's not used in the improver_suite, so should be OK to go straight to master if that's OK with you.

**Comment by arh89** [comment]
@cgsandford Fine with me. Merge away whenever you're happy.

File: lib/improver/cli/extend_radar_mask.py (hunk 1 above)
Comment (neilCrosswaite) [reply lib/improver/cli/extend_radar_mask.py]: Done

**Pull request merged**